index.js

```diff
@@ -385,8 +385,13 @@
   //. > S.pipe([R.inc, Math.sqrt, R.dec])(99)
   //. 9
   //. ```
+  var uncurriedCompose = function(f, g) {
+    return function(x) {
+      return f(g(x));
+    };
+  };
   S.pipe = def('pipe', [List, a], function(fs, x) {
-    return R.reduceRight(compose, I, fs)(x);
+    return R.reduceRight(uncurriedCompose, I, fs)(x);
   });
```

Comment:
Let's define the function in situ. Also, could we switch the application order and use `R.reduce` rather than `R.reduceRight`?